app/mailers/distribution_mailer.rb

```diff
@@ -6,6 +6,8 @@ class DistributionMailer < ApplicationMailer
   #   en.distribution_mailer.partner_mailer.subject
   #
   def partner_mailer(current_organization, distribution, subject)
+    return if distribution.past?
```

Comment:
Not sure whether I want the guard clause in the mailer or in the mailer job. hmmm

Reply:
I like it here in the mailer, so that way if you ever call it from another job, or somewhere else, the guard does its job and prevents mail from going out.